### ISSUE TITLE
Any keypress from editable elements does not set keyboard navigation mode

### ISSUE THREAD

**Issue by ling1726** [microsoft/keyborg#50]
https://github.com/microsoft/keyborg/blob/197b994e8cb7a887a19e025c01f49db802d1e0e0/src/Keyborg.ts#L244-L253

The current keyboard navigation check is done so that any keypress (even ones explicitly set by the user) will not trigger keyboard navigation mode. This results in keys like `Tab` not triggering keyboard navigation mode:

Repro: https://codesandbox.io/p/sandbox/laughing-julien-fzkln2?file=%2Fsrc%2Fexample.tsx
1. Click on the input
2. Press tab
3. First button does not have a focus outline
